### DISCUSSION TITLE
Does fontsource not work with Angular?

### DISCUSSION THREAD

**Discussion by GemmaAnn** [fontsource/fontsource#485] [Q&A] [answered]
Does fontsource not work with Angular?

**Comment by jwr1** [comment, marked as answer]
It does work with Angular, Fontsource was created to be framework agnostic. You can either put something like `../../node_modules/@fontsource/alex-brush/400.css` under `styleUrls` in your `app.component.ts` file, or in one of your CSS files you can put `@import '~@fontsource/alex-brush/400.css';` at the top. The second option is preferred as it is not a relative import.
